seqr/views/apis/summary_data_api.py

```diff
@@ -629,11 +629,11 @@ def _load_aip_full_report_data(data: dict, user: User):
         saved_variant_map.update(new_variants_from_search)
 
     # Add the aip_permissive tag to all variants
-    aip_tag_type = VariantTagType.objects.get(name='AIP-permissive', project=None)
+    aip_tag_type = VariantTagType.objects.get(name='Talos-permissive', project=None)
```

Comment:
Is it worth also changing the tag type/name to be talos, or would that mean the previously added tags need to be re-made?

Reply:
I went back and forth on this. There is a set of AIP tagging code that Hana put in up stream and I decided not to rename all of that in this PR. This PR only renames the CPG specific additions and does the minimal updates to support the changes to the Talos output. We should put in a clean PR to upstream that renames the original AIP methods.